### DISCUSSION TITLE
Exporting a Gatsby.js website that's already hosted online

### DISCUSSION THREAD

**Discussion by andrew-je** [gatsbyjs/gatsby#38118] [Help] [answered]
_Extremely random situation_, I have lost access to my old GitHub account where I had the code for this website: http://themooringscottage.com

The website is still live and I want to rebuild it again using Gatsby.js. Any tips for exporting the site and using it as a base? 

I tried using 'site sucker' to download the site but its a big mess. Thanks!

**Comment by 808vita** [comment, marked as answer]
maybe tryout the sources tab to in developer tools 
![image](https://github.com/gatsbyjs/gatsby/assets/97225946/964334dd-ee78-4206-98f5-40e3984555e8)
